app/src/main/java/me/hufman/androidautoidrive/phoneui/SpotifyAuthorizationActivity.kt

```diff
@@ -63,7 +63,8 @@ class SpotifyAuthorizationActivity: Activity() {
 			SpotifyScope.USER_LIBRARY_READ.uri,
 			SpotifyScope.PLAYLIST_MODIFY_PRIVATE.uri,
 			SpotifyScope.PLAYLIST_READ_PRIVATE.uri,
-			SpotifyScope.UGC_IMAGE_UPLOAD.uri)
+			SpotifyScope.UGC_IMAGE_UPLOAD.uri,
+			SpotifyScope.USER_READ_PRIVATE.uri)
```

Comment:
From what I've read, this permission is only needed if we wanted to load the Country/Market from the user's account. Passing in FROM_TOKEN doesn't seem to need it, and it worked fine when I tested just now without this permission.

Reply:
I was having issues with authenticating Spotify - during the AAIDrive getting started wizard, I would authenticate and see an error for "Insufficient client scope" and the AAIDrive would prompt me again to try again.

I wish I could find the obscure documentation about this that I found last time... I'll try it without the permission again shortly and loop back.

Reply:
update: so removing the scope and installing a new build - everything worked fine. However, when I clear data on the build and try the authentication from scratch, that's when I have errors authenticating (insufficient scope)

` E/SpotifyWebApi: Failed to get search results for query query due to unauthorized error with the message: Received Status Code 403. Error cause: Insufficient client scope`

Reply:
Mmmm yes, I just found that too =\
However, while switching the config around, this search check does automatically prompt the user to re-authenticate to include the new scope, which is ideal!